pkgs/development/libraries/sqlite/default.nix

```diff
@@ -58,6 +62,10 @@ stdenv.mkDerivation rec {
 
   # Test for features which may not be available at compile time
   preBuild = ''
+    mkdir -p $doc/share/doc
```

Comment:
These should be in `postInstall`

Reply:
Ah good catch, I forgot that I had to move it there after debugging